recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -3,7 +3,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 SRC_URI += "file://boot.txt \
 	   "
 
-DEPENDS += "arm-trusted-firmware u-boot-tools-native"
+DEPENDS += "arm-trusted-firmware u-boot-mkimage-native"
```

Comment:
Hmmm....

I don't want to change this to something that doesn't work with master, but I see the problem here. Is there any better way to do this?

Reply:
When you build on the thud branch the u-boot-tools-native does not exist.